auth/utils/file-reading-helpers.ts

```diff
@@ -59,11 +62,33 @@ export function fileExistsAtPath(filepath): boolean {
   return false;
 }
 
-export function constructFilepath(filepath): string {
+export function constructFilepath(filepath: string): string {
   // ensure filepath includes the filename
-  if (!filepath.endsWith(filename)) {
-    filepath = path.join(filepath, filename);
+  if (!filepath.endsWith(defaultCredsFilename)) {
+    filepath = path.join(filepath, defaultCredsFilename);
   }
 
   return filepath;
 }
+
+export function readCrTokenFile(filepath: string): string {
+  if (!fs.existsSync) {
+    return '';
+  }
+
+  let token: string = '';
+  if (fileExistsAtPath(filepath)) {
+    token = fs.readFileSync(filepath, 'utf8');
+    logger.debug(`Successfully read CR token from file: ${filepath}`);
+  } else {
+    logger.error(`Expected to find CR token file but the file does not exist: ${filepath}`);
+    throw new Error(`File does not exist: ${filepath}`);
```

Comment:
Does the `fileExistsAtPath` catch permission issues? If not these messages could be misleading. What do you think?

Reply:
No, permission issues will throw an error within the actual `readFileSync` method and the error shown will say "permission denied". So there shouldn't be an issue with confusing error messages here